core/controllers/collection_editor_test.py

```diff
@@ -38,6 +39,17 @@ def setUp(self):
 
         self.set_admins([self.ADMIN_USERNAME])
 
+        self.owner_role = user_services.get_user_role_from_id(
+            self.owner_id)
+        self.owner = user_services.UserActionsInfo(
```

Comment:
One question. If the second arg here is _always_ obtained from user_services.get_user_role_from_id(user_id), can we just avoid passing the second arg to the UserActionsInfo constructor and keep the get_user_role_from_id() part within the initialization of that object? It seems a bit unnecessary to repeat that logic externally every time.